utils/cliutils/commandsflags.go

```diff
@@ -1065,11 +1065,11 @@ var flagsMap = map[string]cli.Flag{
 	},
 	watches: cli.StringFlag{
 		Name:  watches,
-		Usage: "[Optional] A comma seperated list of Xray's watches, that will determine Xray's violations creation. ` `",
+		Usage: "[Optional] A comma seperated list of Xray watches, that to determine Xray's violations creation. ` `",
```

Comment:
A comma seperated list of Xray watches, that to determine Xray's violations creation.
-->
A comma separated list of Xray watches, to determine Xray's violations creation.